rocon_remocon/src/rocon_remocon/interactive_client.py

```diff
@@ -104,9 +104,21 @@ def _connect(self, rocon_master_name="", ros_master_uri="http://localhost:11311"
         # for the remocon-role manager interaction anyway:
         rospy.init_node(self.name, disable_signals=True)
         try:
-            get_interactions_service_name = rocon_python_comms.find_service('rocon_interaction_msgs/GetInteractions', timeout=rospy.rostime.Duration(5.0), unique=True)
-            get_roles_service_name = rocon_python_comms.find_service('rocon_interaction_msgs/GetRoles', timeout=rospy.rostime.Duration(5.0), unique=True)
-            request_interaction_service_name = rocon_python_comms.find_service('rocon_interaction_msgs/RequestInteraction', timeout=rospy.rostime.Duration(5.0), unique=True)
+            console.logdebug("Interactive Client : Get interactions service Handle")
+            interactions_namespace = rocon_python_comms.find_namespace('get_interactions', 'rocon_interaction_msgs/GetInteractions')
+
+            remocon_services = {}
+            remocon_services['get_interactions'] = interactions_namespace + '/' + 'get_interactions'
```

Comment:
How about making `get_interactions`, `get_roles`, and `request_interaction` as class constant value?